smarts/benchmark/driving_smarts/v2022/config.yaml

```diff
@@ -31,16 +31,16 @@ benchmark:
         - scenarios/sumo/straight/3lane_overtake_agents_1
       kwargs:
         seed: 42
-    # bubble: # reserved for bubble env
-    #   loc: "smarts.env:driving-smarts-competition-bubble-env-v0"
-      # scenarios:
-      #   - 6
-      #   - 4
-      #   - 42
+    # bubble:
+    #   loc: "smarts.env:driving-smarts-bubble-v2022"
+    #   scenarios:
+    #     - 6
+    #     - 4
+    #     - 42
     #   kwargs:
-    # naturalistic: # reserved for driving_smarts_competition_naturalistic-v0
-    #   loc: "smarts.env:driving_smarts_competition_naturalistic-v0"
+    # naturalistic:
+    #   loc: "smarts.env:driving-smarts-naturalistic-v2022"
```

Comment:
This reminds me that we should add a standard way to deal with the naturalistic scenarios.

Reply:
Agreed.